config/locales/en/errors.yml

```diff
@@ -50,3 +50,4 @@ en:
   staff_response:
     errors:
       must_reject_or_accept_visit: "A visit must either be accepted by choosing a date or at least one rejection reason"
+      lead_visitor_not_on_contact_list: The lead visitor is not on the contact list
```

Comment:
@StupidCodeFactory Will this be a specific rejection reason in the rejection_decorator and a rejection email sent? We don't seem to have a lead visitor rejection email.